test/Microsoft.AspNetCore.TestHost.Tests/ResponseFeatureTests.cs

```diff
@@ -40,5 +41,28 @@ public void OnStarting_ThrowsWhenHasStarted()
                 }, state: "string");
             });
         }
+
+        [Fact]
+        public void StatusCode_ThrowsWhenHasStarted()
```

Comment:
Suggest adding a similar test for `ReasonPhrase`.

Reply:
I covered reason phrase in this test